Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -343,18 +343,33 @@ extension NavigationMapView {
     /**
      Given a congestion level, return its associated color.
      */
-    func congestionColor(for congestionLevel: String?) -> UIColor {
-        switch congestionLevel {
-        case "low":
-            return trafficLowColor
-        case "moderate":
-            return trafficModerateColor
-        case "heavy":
-            return trafficHeavyColor
-        case "severe":
-            return trafficSevereColor
-        default:
-            return trafficUnknownColor
+    func congestionColor(for congestionLevel: String?, isMain: Bool) -> UIColor {
+        if isMain {
+            switch congestionLevel {
+            case "low":
+                return trafficLowColor
```

Comment:
As I mentioned earlier something like `return isMain ? trafficLowColor : alternativeTrafficLowColor` would be more descriptive/less error prone in my opinion.

Reply:
Thanks for your suggestion! I'll update this part.